scripts/kconfig/kconfigfunctions.py

```diff
@@ -47,6 +62,8 @@ def dt_int_val(kconf, _, name, unit=None):
     if doc_mode or name not in dt_defines:
         return "0"
 
+    print("WARNING: dt_int_val is deprecated.")
```

Comment:
Might be nice to add a `warn()` helper that sends messages to stderr:

```python
def warn(msg):
    print(msg, file=sys.stderr)
```

Reply:
This might be better, so that it gives the location as well (like `err()`):

```python
def warn(kconf, msg):
    print("{}:{}: warning: {}".format(kconf.filename, kconf.linenr, msg))
```

Reply:
How about we do the warn cleanup post merge of your kconfig change.

Reply:
Could maybe do just the first `warn()` version for now.